x-pack/plugins/task_manager/server/task_claimers/strategy_default.test.ts

```diff
@@ -351,6 +352,28 @@ describe('TaskClaiming', () => {
         },
       });
       expect(sort).toMatchObject([
+        {
```

Comment:
ugh!  scripted sort!

I think the only way to avoid would be to set the priority in the task SO's which would require a migration.  Given the expected size of the things being sorted though (hundreds, maybe) doesn't seem worth it.  

Also, presumably this will change once we change the task claimer from mark/query to query/mget/updated (we'll get more tasks than we can run (in case some we get end up getting claimed elsewhere) and can do a local sort on priority).

Reply:
Yes! I'm hoping that the switch to query/mget/updated will make this better.